src/bfio/ts_backends.py

```diff
@@ -24,19 +26,116 @@ class TsOmeTiffReader(bfio.base_classes.TSAbstractReader):
     def __init__(self, frontend):
         super().__init__(frontend)
 
-        self.logger.debug("__init__(): Initializing _rdr (TSTiffReader)...")
-        self._rdr = TSTiffReader(str(self.frontend._file_path))
+        self.logger.debug("__init__(): Initializing _rdr (TSReader)...")
+        extension = "".join(self.frontend._file_path.suffixes)
```

Comment:
There are no getters for this object at the moment. Since everything is public in Python and this is a low-level library implementation code (no user API has direct access to `frontend`), I opted not to add another layer of indirection.